src/OpenSearch.OpenSearch.Xunit/XunitPlumbing/SkippingTestCase.cs

```diff
@@ -45,9 +45,11 @@ public SkippingTestCase()
 		/// <param name="testMethod">The test method this test case belongs to.</param>
 		/// <param name="testMethodArguments">The arguments for the test method.</param>
 		public SkippingTestCase(string skipReason, ITestMethod testMethod, object[] testMethodArguments = null)
-			: base(TestMethodDisplay.ClassAndMethod, testMethod, testMethodArguments) =>
+			: base(TestMethodDisplay.ClassAndMethod, TestMethodDisplayOptions.None, testMethod, testMethodArguments) =>
 			SkipReason = skipReason ?? "skipped";
 
+		public int Timeout => 0;
```

Comment:
This looks new, why was it added?

Reply:
`xunit` was updated and interface `TestMethodTestCase` got a new method `Timeout` which should be overloaded.